docs/snippets/cpu/multi_threading.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/runtime/core.hpp>
+
+#include "openvino/runtime/intel_cpu/properties.hpp"
+
+int main() {
+    try {
+        std::string modelPath = "model.xml";
+        std::string device = "CPU";
+        ov::AnyMap config;
+        ov::Core core;
+        core.set_property(ov::inference_num_threads(1));
+        auto model = core.read_model(modelPath);
+        //! [ov:intel_cpu:multi_threading:part0]
+        // Use one processor for inference
+        auto compiled_model_1 = core.compile_model(model, device, ov::inference_num_threads(1));
+
+        // Use processors of Efficient-cores for inference on hybrid platform
+        auto compiled_model_2 = core.compile_model(model, device, ov::hint::scheduling_core_type(ECORE_ONLY));
```

Comment:
How to judge these properties take effect in code level?

Reply:
user can query `ov::hint::scheduling_core_type` in ExecutableNetwork